1-Draft/RFCnnnn-PowerShell-Configuration.md

```diff
@@ -54,9 +54,19 @@ Scheme | Windows | Unix
 GPO -> Computer Policy | HKLM\Software\Policies\PowerShellCore | /etc/powershell.config.json
 GPO -> User Policy | HKCU\Software\Policies\PowerShellCore | See `Comment A` below
 File -> Computer-Wide | %ProgramFiles%/PowerShell/powershell.config.json | /opt/Microsoft/powershell/powershell.config.json
-File -> User-Wide | %APPDATA%/powershell.config.json | ~/powershell.config.json
-File -> Application-Wide | $apphome/powershell.config.json | $apphome/powershell.config.json
 File -> Application-Startup | pwsh -settingsfile `somepath/powershell.config.json` | pwsh -settingsfile `somepath/powershell.config.json`
+File -> User-Wide | %APPDATA%/powershell.config.json | %XDG_CONFIG_HOME%/powershell.config.json
+File -> Application-Wide | $apphome/powershell.config.json | $apphome/powershell.config.json
+
+Defaults:
+
+`%APPDATA%` - `C:\Users\useraccount\AppData\Roaming`
+
+`%XDG_CONFIG_HOME%` - `HOME/.config`
+
+#### Parameter `-settingsfile`
+
+With `-settingsfile` parameter users can assign custom settings from the config file and overwrite user-wide and application-wide settings. Only users with elevated rights can overwrite computer-wide and user policy settings.
```

Comment:
When we are in lockdown mode, even elevated users should not be able to overwrite computer-wide settings

Reply:
> Only users with elevated rights can overwrite computer-wide and user policy settings.

I do not know how to formulate it right at the same time for Windows (GPO) and for Unix (only files).
In fact, the administrator always has the ability to change any setting. 
Maybe remove the sentence at all?